utils/rpms/daos.spec

```diff
@@ -414,6 +414,12 @@ getent group daos_metrics >/dev/null || groupadd -r daos_metrics
 getent group daos_server >/dev/null || groupadd -r daos_server
 getent group daos_daemons >/dev/null || groupadd -r daos_daemons
 getent passwd daos_server >/dev/null || useradd -s /sbin/nologin -r -g daos_server -G daos_metrics,daos_daemons daos_server
+# Ensure /var/log/daos exists
+if [ ! -d /var/log/daos ]; then
+    mkdir -p /var/log/daos
+    chown daos_server.daos_daemons /var/log/daos
+    chmod 775 /var/log/daos
```

Comment:
Might be worth making "/var/log/daos" a variable set at the top of the specfile, so it's only defined in one place. I see it's used elsewhere in the specfile as well.